headlamp-plugin/Makefile

```
@@ -0,0 +1,63 @@
+test: test-unit
+
+.PHONY: test-e2e
+test-e2e:
+	@echo "No e2e tests configured yet"
+
+.PHONY: build
+build:
+	npm run build
+
+.PHONY: images
+images:
```

checkmake (0.3.2)

[warning] Target body for "images" exceeds allowed length of 5 lines (6).

(maxbodylength)